### PR TITLE
chore(hubble): Enhance code style in hubble

### PR DESCRIPTION
**Pull request by returnToInnocence**
## Purpose of the PR

- subtask of #591   

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [x] Need tests and can be verified as follows:
    - Check against the [HugeGraph Code Style Guide](https://github.com/apache/incubator-hugegraph/wiki/HugeGraph-Code-Style-Guide)

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-hubble/hubble-fe/node/node.exe
Comment (imbajin) [inline hugegraph-hubble/hubble-fe/node/node.exe:1]: unexpected binary file, could check the **updated/new files** before submit PR

Comment (returnToInnocence) [reply hugegraph-hubble/hubble-fe/node/node.exe:1]: Sorry for my negligence. I have corrected it now🙈

File: pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
     </modules>
 
     <properties>
-        <revision>1.2.0</revision>
+        <revision>1.5.0</revision>
```
[the remark this answers is not in the document]
Comment (returnToInnocence) [reply pom.xml:101]: I mistakenly mixed the unified requirements of pd/store version with hubble. I will modify it immediately🫡

**Review by imbajin — APPROVED** [review]
Thanks~

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**